app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+app.use(express.json())
+const { fetchAllArticles } = require('./controllers/fetchAllArticlesController')
+const { handles500Errors, handles400Errors, handle404nonExistentPath } = require('./handlesErrors')
+
+
+app.get("/api/articles", fetchAllArticles)
+app.use(handle404nonExistentPath)
+app.use(handles500Errors)
+app.use(handles400Errors)
+
+
```

Comment:
Here we should add a default 404 handler (have a look at using a path /* to catch general invalid paths) - this should be in app.js - not the error file. The error file is only for psql errors (ones where you have error codes thrown) or customer errors (promise.rejects thrown)